src/universal/modules/teamDashboard/components/ArchiveTeam/ArchiveTeamForm.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React, {Component} from 'react';
+import {reduxForm, Field} from 'redux-form';
+import InputField from 'universal/components/InputField/InputField';
+
+type Props = {
+  handleFormBlur: () => any,
+  handleFormSubmit: () => any,
+  handleSubmit: (fn: () => any) => any,
+  teamName: string
+};
+
+class ArchiveTeamForm extends Component<Props> {
+  validate = (value) => (
+    value !== this.props.teamName
+      ? 'The team name entered was incorrect'
```

Comment:
Small thing. I would add a period at the end of the input error message since this is a sentence.